### ISSUE TITLE
latest Tweetdeck update broke BTD (Uncaught TypeError: $ is not a function)

### ISSUE THREAD

**Issue by Mohamed3on** [eramdam/BetterTweetDeck#357]
Error in the console:
```
Uncaught TypeError: $ is not a function
    at t.default (chrome-extension://micblkellenpbfapmcpcfhcoeohhnpob/js/inject.js:11)
    at Object.347 (chrome-extension://micblkellenpbfapmcpcfhcoeohhnpob/js/inject.js:18)
    at n (chrome-extension://micblkellenpbfapmcpcfhcoeohhnpob/js/inject.js:1)
    at 0 (chrome-extension://micblkellenpbfapmcpcfhcoeohhnpob/js/inject.js:1)
    at chrome-extension://micblkellenpbfapmcpcfhcoeohhnpob/js/inject.js:1
```
Problematic line:
https://github.com/eramdam/BetterTweetDeck/blob/411402766bf5feb0b11d4b62d68dec1f5666cf00/src/js/util/ame.js#L220
This is what `$` resolves to
![image](https://user-images.githubusercontent.com/12295159/42159321-a3359590-7df3-11e8-85f0-24a31c440b28.png)

So it's overwritten?

**Comment by NuroDev** [comment]
This was already mention in a small thread the other day: [https://twitter.com/BetterTDeck/status/1013252407180984321](https://twitter.com/BetterTDeck/status/1013252407180984321)

**TLDR;** Tweetdeck has made some drastic changes that BTD was already made aware of prior to the update this morning and a patch/fix is already being worked on. Hopefully should get an update out soon.

**Comment by Mohamed3on** [comment]
Hmm, the thread says the update prevents BTD from blowing up, but I'm on that version, and BTD still fails to load?

**Comment by pixeldesu** [comment]
@NuroDev The update is already out, the update is what caused this error now.

We're overwriting $ in all cases now, and over the weekend our search function returned the proper jQuery module, but it seems that with a recent deploy the module order changed and something else is now in place where our reference to jQuery was before.

We need to find a search query that always returns us just the jQuery instance, not any other things that might slip in.

**Comment by eramdam** [comment]
Gonna close this as it should be fixed with 3.7.8
